index.js

```diff
@@ -7,14 +7,24 @@ const prettyMs = require('pretty-ms');
 
 const getMetaTag = (html, property) => {
 	const regex = new RegExp(`<meta[^>]*property=["|']${property}["|'][^>]*>`, 'i');
+	const executedRegex = regex.exec(html);
 
-	return regex.exec(html)[0];
+	if (!executedRegex) {
+		throw new Error(`Missing ${property}`);
+	}
+
+	return executedRegex[0];
 };
 
-const getMetaTagContent = metaTagHtml => {
+const getMetaTagContent = (metaTagHtml, property) => {
```

Comment:
Still no need to be passing property in here.

We can grab it from the `metaTagHtml`. Or even just dump the entire contents of `metaTagHtml` to the console. It's probably not a lot of text.